executor/aggregate.go

```diff
@@ -73,7 +72,7 @@ type HashAggFinalWorker struct {
 
 	rowBuffer           []types.Datum
 	mutableRow          chunk.MutRow
-	aggCtxsMap          aggCtxsMapper
+	partialResultMap    aggPartialResultMapper
 	groupSet            *mvmap.MVMap
```

Comment:
seems `groupSet` can be declared as `map[string]struct{}`, or just use `executor.aggfuncs.stringSet`?

We can leave it to the future PRs.